backend/codesirius/problems/models/__init__.py

```
@@ -1,7 +1,15 @@
 from problems.models.submission import Submission
 from problems.models.tag import Tag
 
-__all__ = ["Language", "Tag", "Problem", "ReferenceSolution", "Submission"]
+__all__ = [
+    "Language",
+    "Tag",
+    "Problem",
+    "ReferenceSolution",
+    "Submission",
+    "ExecutionConstraint",
+]
```

Ruff (0.8.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)